### DISCUSSION TITLE
Update URLs and logos with X design (former Twitter)

### DISCUSSION THREAD

**Discussion by aescanes** [monkeytypegame/monkeytype#5478] [Ideas]
Update URLs and logos with X design (former Twitter). Now that we can use fontawesome-6, this change should be easy.
I can create a fork and work on this if you guys agree.

https://fontawesome.com/icons/x-twitter?f=brands&s=solid

**Comment by Whydidthechickencrosstheroad** [comment]
Yeah, I noticed that it was not updated a little while since the update logo with X (or Twitter I should say)

**Reply by unrealapex** [reply]
I'll shoot a PR to fix this.

**Reply by Whydidthechickencrosstheroad** [reply]
It still hasn't been finished

**Reply by unrealapex** [reply]
#5541

Discussing with Miodec, we decided that it was best to continue referring to X as Twitter on the site, but the all the links are now updated to [x.com](https://x.com). Also FYI the current version of Font Awesome that Monkeytype uses doesn't have X's icon.
